### ISSUE TITLE
[a11y] Product page - Missing `alt` attribute on images

### ISSUE THREAD

**Issue by g123k** [openfoodfacts/openfoodfacts-server#10292]
Hi everyone,

To improve accessibility on the website, we have some "simple" fixes to implement.
[On the product page](https://world.openfoodfacts.org/product/3274080005003/eau-de-source-cristaline), we have several elements without an `alt` attribute:

- [ ] We have labels, such as "Triman"
- [ ] The Nutri-score logo
- [ ] The Ecoscore logo
- [ ] The NOVA logo
